source/codegen/metadata/nidmm/functions.py

```diff
@@ -1495,6 +1616,28 @@
         ],
         'returns': 'ViStatus'
     },
+    "InitExtCal": {
+        "init_method" : True,
+        "codegen-method" : None,
```

Comment:
Oh, I missed this in the last update. This should be "codegen_method" not "codegen-method". That's why the InitExtCal is still generating.

Reply:
Oops, fixed that. But it is still present in the library files.

Reply:
Oh, I think that's fine / a bit of oversight in the codegen to not honor the codegen_method in that case. 

I think it's fine to go in as is in any case since we'll be using those library file methods in the upcoming change anyway.